app/services/sns-success-updater.js

```diff
@@ -21,7 +21,8 @@ function messageFrom (staffCode, availableHours) {
     },
     personReference: {
       identifiers: [
-        { type: 'staffCode', value: { staffCode } }
```

Comment:
this didn't match the event definition. This would create a json like `{ type:  'staffCode', value: { staffCode: 'STAFF1'}}` where we want the value to be a string and not an object